eo-maven-plugin/src/test/java/org/eolang/maven/objectionary/OyLocalTest.java

```diff
@@ -45,20 +42,22 @@ final class OyLocalTest {
     @Test
     void resolvesObjectInLocalStorage(@TempDir final Path path) throws Exception {
         final String content = "[] > main\n";
-        new LengthOf(
-            new TeeInput(
-                new InputOf(content),
-                new OutputTo(
-                    path.resolve("pulled/master/org/example/main.eo")
-                )
-            )
-        ).value();
+        new Home(path).save(
```

Comment:
@MikhailLipanin It's better to split the test to 2 cases:
1. check for `get`
2. check for `contains`

Reply:
done